src/routers/alpha-router/alpha-router.ts

```diff
@@ -407,6 +407,15 @@ export class AlphaRouter
           break;
         case ChainId.ARBITRUM_ONE:
         case ChainId.ARBITRUM_RINKEBY:
+
+        /*
+          TODO understand the reason behind the "out of gas" error we get if we
+          don't have this here due to low gas limit or low gas price
+          The following gas limits seem to be sufficient for transactions on Celo.
+          Optimism & default limits caused the 'out of gas' error
+        */
+        case ChainId.CELO:
```

Comment:
Recommend changing multicall chunk for the fallback to 2, as I noticed that `./bin/cli quote --tokenIn 0xF194afDf50B03e69Bd7D057c1Aa9e10c9954E4C9 --tokenOut 0x874069Fa1Eb16D44d622F2e0Ca25eeA172369bC1 --amount 1 --exactIn --protocols v3 --recipient 0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B --minSplits 1 --router alpha --chainId 44787 --debug` doesn't work with these settings due to out of gas

Reply:
Is this comment resolved?